pkg/datapath/linux/node.go

```diff
@@ -295,15 +295,18 @@ func (n *linuxNodeHandler) deleteDirectRoute(CIDR *cidr.CIDR, nodeIP net.IP) {
 	}
 }
 
-// createNodeRoute creates a route that points the specified prefix to the host
-// device via the router IP
+// createNodeRouteSpec creates a route spec that points the specified prefix to the host
+// device via the router IP. The route is configured with a computed MTU for non-local nodes.
```

Comment:
nit: Is this comment in sync with the parameters of the function? Seems like it's assuming `isLocalNode` == `false`, because if it's true then the route is not configured with an MTU now.

Reply:
I'm not sure I follow. A route is configured with an MTU for "non-local" nodes (i.e., `isLocalNode` is false). I can make the description more explicit.

Reply:
oh, I think I just misread it the first time. Looking again it LGTM. :+1: